Frameworks/SPMySQLFramework/SPMySQLFramework.xcodeproj/project.pbxproj

```diff
@@ -615,6 +615,7 @@
 			hasScannedForEncodings = 1;
 			knownRegions = (
 				en,
+				Base,
```

Comment:
Intentional?

Reply:
Yes! Just to fix warning, not actually using it